documentation/src/router.ts

```diff
@@ -35,6 +36,8 @@ class DocumentationRouter extends Router {
     }
 
     public go(pathname: string): boolean {
+        const docs = document.querySelector('docs-page') as Page;
+        docs.resetScroll();
```

Comment:
Make sure we go back to the top when the page is changed. Later, we'll probably need to expand on this functionality to programmatically visit anchor content when present.